tasks/elasticsearch-plugins.yml

```diff
@@ -46,18 +46,20 @@
   environment:
     CONF_DIR: "{{ conf_dir }}"
     ES_INCLUDE: "{{ instance_default_file }}"
+    ES_ENV_FILE: "{{ instance_default_file }}"
 
 - name: Install elasticsearch plugins
-  command: "{{es_home}}/bin/elasticsearch-plugin install {{ item.plugin }} --batch --silent"
+  command: "{{es_home}}/bin/elasticsearch-plugin install {{ item.url | default(item.plugin) }} --batch --silent"
   register: plugin_installed
```

Comment:
although the use of `url` is clearer, this does currently work if the user sets the plugin to a url explicitly. Given it doesn't break backwards compatibility, and its cleaner, happy to merge. @barryib